geode-lucene/src/test/java/org/apache/geode/cache/lucene/internal/LuceneIndexCreationProfileDUnitTest.java

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I was expecting this test to fail as i am trying to create two indexes with the same name but different fields , but it was able to get to executing the Lucene queries and two different indexes were created in two different members. In my opinion we should prevent the execution to continue if there is an exception in the cache profile validation.
```
 @Test
  @Parameters(method = "getListOfRegionTestTypes")
  public void dropAndRecreateIndex(RegionTestableType regionTestType) throws Exception {
    SerializableRunnableIF createIndex = () -> {
      LuceneService luceneService = LuceneServiceProvider.get(getCache());
      luceneService.createIndexFactory().addField("text").create(INDEX_NAME, REGION_NAME);
    };
    dataStore1.invoke(() -> initDataStore(() ->{}, regionTestType));
    dataStore2.invoke(() -> initDataStore(() -> {}, regionTestType));
    accessor.invoke(() -> initAccessor(() ->{}, regionTestType));

    putDataInRegion(accessor);

    // re-index stored data
    AsyncInvocation ai1 = dataStore1.invokeAsync(() -> {
      LuceneService luceneService = LuceneServiceProvider.get(getCache());
      LuceneIndexFactoryImpl indexFactory =
          (LuceneIndexFactoryImpl) luceneService.createIndexFactory().addField("text");
      indexFactory.create(INDEX_NAME, REGION_NAME, true);
    });

    AsyncInvocation ai2 = dataStore2.invokeAsync(() -> {
      LuceneService luceneService = LuceneServiceProvider.get(getCache());
      LuceneIndexFactoryImpl indexFactory =
          (LuceneIndexFactoryImpl) luceneService.createIndexFactory().addField("text2");
      indexFactory.create(INDEX_NAME, REGION_NAME, true);
    });

    ai1.join();
    ai2.join();

    ai1.checkException();
    ai2.checkException();

    waitForFlushBeforeExecuteTextSearch(accessor, 60000);
    executeTextSearch(accessor);
  }
```